make shell script that do 'cargo run' and launch browser(google-chrome)

seiay opened the pull request:
## Description
I made a shell script that can do a 'cargo run'  command and launch a browser(google-chrome).
You can start up the web app only by executing start-browser. sh.

## Test performed
I confirmed the start-up of the web app by `./start-browser.sh`.

seiay committed:
make shell script that do 'cargo run' and launch browser(google-chrome)

asana17 commented on start-browser.sh:
```diff
@@ -0,0 +1,9 @@
+# cargo run
+cargo build
+(
+	sleep 1
```
This expression (`sleep 1`) is OK temporarily, but we need to check target app is running by some method (for example, the `ps` command). You don't need to change it in this PR.

seiay replied:
Thanks!

> but we need to check target app is running by some method (for example, the ps command). 

Ok.

asana17 reviewed (changes requested):
The script name `start-browser.sh` looks like starting only the browser. Something like `run_with_browser.zsh`is better,  I think.

seiay committed:
rename shell script and edit README

seiay committed:
modify README

seiay commented:
> The script name start-browser.sh looks like starting only the browser. Something like run_with_browser.zshis better, I think.

I think so too.
I renamed the script name and also changed README.
